doc/examples/segmentation/plot_floodfill.py

```diff
@@ -142,3 +140,47 @@
 
 fig.tight_layout()
 plt.show()
+
+##############################################################################
+# Flood-fill in HSV space and mask post-processing
+# ------------------------------------------------
+#
+# Since flood fill operates on single-channel images, we transform here the
+# image to the HSV (Hue Saturation Value) space in order to flood pixels of
+# similar hue.
+#
+# In this example we also show that it is possible to post-process the binary
+# mask returned by :func:`skimage.segmentation.flood` thanks to the functions
+# of :mod:`skimage.morphology`.
+
+
+img = data.astronaut()
+img_hsv = color.rgb2hsv(img)
+
+img_hsv_copy = np.copy(img_hsv)
+# Fill hue channel with a different color using flood fill
+_ = flood_fill(img_hsv_copy[..., 0], (313, 160), 0.5,
+               tolerance=0.016, in_place=True)
+
+# Return mask for post-processing
+mask = flood(img_hsv[..., 0], (313, 160), tolerance=0.016)
```

Comment:
I feel this post-processing may unnecessarily complicate the example. And since the flood actually propagates into the areas beyond the suit, showing the patched mask contour may be a bit unfair :sweat_smile: . Do you think we could show here only the `flood_fill` result? Perhaphs, obtained with a higher `tolerance`, e.g. 0.03.

Reply:
I'm of the opposite opinion: since we are doing post-processing on the mask, let's not do the fill — only fill at the end. What do you think @emmanuelle?

Reply:
@jni that would be infinitely better than my suggestion!

Reply:
@emmanuelle I just realised reading the comments that you are showing *both* methods, `flood_fill` and `flood`. I think it feels a bit artificial. What I was suggesting was to just use `flood`, then do `img_hsv[mask, 0] = 0.5` and `img_hsv_copy[mask_post_processed, 0], 0.5`, and show both images. I'll note that you never convert `img_hsv` back to RGB. After `img_hsv[mask, 0] = 0.5` you just never use `img_hsv` again.